alerts/src/commonMain/kotlin/Alerts.kt

```diff
@@ -69,6 +72,17 @@ data class Alert(
             NEGATIVE(CANCEL.value)
         }
     }
+
+    /**
+     * An action that represents an input field in the alert
+     *
+     * @property hint The hint of the input field
+     * @property textWatcher The block to execute when the user edits the text in the input field
+     */
+    data class TextInputAction(
+        val hint: String?,
```

Comment:
I'd name it `textIfEmpty` because `hint` is Android specific name. Also good to have ability to set actual value (as pre fill) to the text field.

Reply:
Maybe `placeholder`?